lib/puppet/util/package/version/rpm.rb

```diff
@@ -0,0 +1,285 @@
+# frozen_string_literal: true
+
+module Version
+  class Error < StandardError; end
```

Comment:
you you need this error class?

Reply:
I think this is already replaced by `ValidationFailure` below. It should be removed.